### PR TITLE
Changed branch, Fixing Media Part

### PR DESCRIPTION
**Pull request by rhemon**

### CODE REVIEW

File: stylesheets/main.css [hunk 1]
```diff
@@ -91,26 +77,21 @@
...
-
 .title {
     padding-top: 30px;
     text-align: center;
-    color:#ffd700;
-    text-shadow: 0px 2px 3px rgba(0,0,0,0.4),
-    0px 7px 6px rgba(0,0,0,0.1),
-    0px 2px 8px rgba(0,0,0,0.1);
+    color: #ffd700;
+    text-shadow: 0px 2px 3px rgba(0, 0, 0, 0.4), 0px 7px 6px rgba(0, 0, 0, 0.1), 0px 2px 8px rgba(0, 0, 0, 0.1);
```
Comment (hemantjadon) [inline stylesheets/main.css]: Here specify in shadows in different lines. (Line length error)

File: index.html [hunk 2]
```diff
@@ -233,50 +233,44 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
             <div class="social-media">
                 <h1 class="white-header">FOSSASIA on social media</h1>
                 <div class="row">
-                    <div class="col-xs-12 col-md-6">
+                    <div class="col-md-6">
                         <!-- Ignore LineLengthBear -->
-                        <iframe src="//www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2Ffossasia&tabs=timeline&width=450&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=true&appId" width="450" height="493"
-                                style="border:none;overflow:hidden" scrolling="no" frameborder="0"
-                                allowTransparency="true"></iframe>
+                        <iframe src="//www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2Ffossasia&amp;tabs=timeline&amp;width=370&amp;height=459&amp;small_header=false&amp;adapt_container_width=true&amp;hide_cover=false&amp;show_facepile=true&amp;appId"
```
Comment (hemantjadon) [inline index.html:291]: Here also

File: index.html [hunk 3]
```diff
@@ -233,50 +233,44 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
...
+                            width="370" height="459" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowtransparency="true"></iframe>
                     </div>
-                    <div class="col-xs-12 col-md-6">
-                        <div class="g-page" data-href="//plus.google.com/+fossasiaorg"
-                             data-width="450"></div>
+                    <div class="col-md-6">
+                        <div class="g-page" data-href="//plus.google.com/+fossasiaorg" data-width="370" data-height="459"></div>
```
Comment (hemantjadon) [inline index.html]: Check here

File: index.html [hunk 4]
```diff
@@ -233,50 +233,44 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
...
                 </div>
                 <div class="row">
-                    <div class="col-xs-12 col-md-6">
-                        <a class="twitter-timeline" data-width="452" data-height="493"
-                           href="//twitter.com/fossasia">Tweets by fossasia</a>
-                    </div>
-                    <div class="col-xs-12 col-md-6">
-                        <div class="github-widget" data-username="fossasia"
-                             style="background:white;margin:auto"></div>
+                    <div class="col-md-6"><a class="twitter-timeline" data-width="370" data-height="459" href="//twitter.com/fossasia">Tweets by
```
Comment (hemantjadon) [inline index.html]: Here also

File: index.html [hunk 5]
```diff
@@ -233,50 +233,44 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
...
+                    <div class="col-md-6">
+                        <div class="github-widget" data-username="fossasia"></div>
                     </div>
                 </div>
-                <div class="row">
-                    <div class="col-xs-12 col-md-6" style="margin-top: 2em; padding: 1em">
+                <div class="last-one">
+                    <div style="margin-top: 2em;">
+                        <div id="gyt">
+                            <div class="g-ytsubscribe" data-channelid="UCQprMsG-raCIMlBudm20iLQ" data-layout="full" data-count="default" height="48px"></div>
```
Comment (hemantjadon) [inline index.html]: Long line

File: index.html [hunk 6]
```diff
@@ -233,50 +233,44 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
...
-                                 data-count="default" style="display:flex; box-shadow: none"></div>
-                            <!-- Ignore LineLengthBear -->
-                            <iframe width="510px" height="310px" src="//www.youtube.com/embed/videoseries?list=PLzZVLecTsGpK039bJFaMsFbYXA6QVPaO5"
-                                    frameborder="0" allowfullscreen style="margin-top: 1em;">
-                            </iframe>
+                        <div class="container">
+                            <div class="row">
+                                <div class="col-xs-12 col-md-12" style="margin-bottom:20px;">
+                                    <div class="embed-responsive embed-responsive-4by3">
+                                        <iframe class="embed-responsive-item" src="//www.youtube.com/embed/videoseries?list=PLzZVLecTsGpK039bJFaMsFbYXA6QVPaO5" frameborder="0" allowfullscreen=""></iframe>
```
Comment (hemantjadon) [inline index.html]: Another long line

File: index.html [hunk 7]
```diff
@@ -233,50 +233,44 @@ <h1 style="color: #ffffff; text-align: center;">MBDyn Logo Designs</h1>
...
+                                    </div>
+                                </div>
+                                <div class="col-xs-12 col-md-12">
+                                    <div id="flickrembed"></div>
+                                    <small style="display: block; text-align: center; margin: 0 auto;"><a href=
```
Comment (hemantjadon) [inline index.html]: Here also a long line

File: index.html (hunk 6 above)
Comment (hemantjadon) [reply index.html]: Also set `allowfullscreen` not `allowfullscreen=""`

File: index.html (hunk 7 above)
Comment (hemantjadon) [reply index.html]: Here are some extra whitespaces don't know where. Please check

File: index.html (hunk 6 above)
Comment (rhemon) [reply index.html]: about long line, how can I separate the src?

**Comment by rhemon** [comment]
@hemantjadon never been this happy seeing a tick

**Comment by hemantjadon** [comment]
Link to gh-pages for review please

**Comment by rhemon** [comment]
https://rhemon.github.io/gci16.fossasia.org this here seems fine, but I don't know why my localhost is not showing up properly

**Comment by rhemon** [comment]
there seems to be conflict with this branch and my gh-pages branch, give me some time to fix those

**Comment by hemantjadon** [comment]
I have added more work needed label

**Comment by rhemon** [comment]
ok thanks, if I can't fix it now I will continue tomorrow morning

**Comment by rhemon** [comment]
@hemantjadon this should be fine now : https://rhemon.github.io/gci16.fossasia.org/
have a look plesae

**Comment by hemantjadon** [comment]
Please squash the commits to 1 and name it something verbose.

**Comment by rhemon** [comment]
I knew it, it just has to be there

**Comment by rhemon** [comment]
Fixing the conflict, give me 5 mins

**Comment by rhemon** [comment]
i don't know why the commit name didn't get changed, wasn't `r` supposed to do that? now that I try rebasing it shows me all these are other commits, and I can even see my one

**Pull request merged**